src/hyperloglog.c

```diff
@@ -13,6 +13,10 @@
 #include <stdint.h>
 #include <math.h>
 
+#if defined (HAVE_AVX2)
```

Comment:
usually we just use `#ifdef` in redis code
```suggestion
#ifdef HAVE_AVX2
```